operators/src/pro/cache/cache_chunks.rs

```diff
@@ -186,11 +174,28 @@ where
         // So the query is not updated.
         Ok(())
     }
+
+    fn cache_element_hit(&self, query: &Self::Query) -> bool {
+        let temporal_hit = if let Some(time_bounds) = self.time_interval {
+            time_bounds == query.time_interval || time_bounds.intersects(&query.time_interval)
+        } else {
+            false // TODO: should we return true here?
+        };
+
+        let spatial_hit = if let Some(spatial_bounds) = self.spatial_bounds {
+            spatial_bounds == query.spatial_bounds
+                || spatial_bounds.intersects_bbox(&query.spatial_bounds)
+        } else {
+            !G::IS_GEOMETRY // return true if the geometry is not spatial
```

Comment:
isn't the only case where spatial_bounds are None, if the collection is DataCollection? And DataCollection is the only one where IS_GEMOTRY is false right? So we could just say true here. Because in a DataCollection, every  feature is valid everywhere and thus contained in any element, regardless of the spatial bounds.

Reply:
Besides the DataCollection it is empty if the collection is empty. In both cases it must return true. It is filtered in later step.

Reply:
[75cf335](https://github.com/geo-engine/geoengine/pull/864/commits/75cf335a877d4817e8b2116cbb12990a519fff5a)